lib/shared_print/deprecation_record.rb

```diff
@@ -77,8 +79,8 @@ def org_commitments
     # If commitments contain deprecated ones, add to @err and remove them.
     # Return only non-deprecated
     def reject_deprecated
-      already_deprecated = org_commitments.select(&:deprecated?)
-      @org_commitments.reject!(&:deprecated?)
+      already_deprecated = org_commitments&.select(&:deprecated?)
```

Comment:
It looks like `org_commitments` could return nil if there was a previous error; I think maybe some of my confusion here is that it's trying to follow through parts of the process if there was an error? Is there a reason to do that rather than baling out after the first error?

Reply:
No more @err, just raising at the first sign of trouble.